src/pages/PrivateNotes/PrivateNotesViewPage.js

```diff
@@ -71,33 +70,28 @@ function PrivateNotesViewPage({route, personalDetailsList, session, report}) {
             includeSafeAreaPaddingBottom={false}
             testID={PrivateNotesViewPage.displayName}
         >
-            <FullPageNotFoundView
-                shouldShow={_.isEmpty(report) || _.isEmpty(report.privateNotes) || !_.has(report, ['privateNotes', route.params.accountID, 'note']) || ReportUtils.isArchivedRoom(report)}
-                subtitleKey="privateNotes.notesUnavailable"
```

Comment:
Looks like we're also not displaying this subtitle anymore

Reply:
@MonilBhavsar Resolved merge conflicts and tested scenarios.

By the way, on merge, I found out that in recent migration of `withReportOrNotFound` migration to tsx, default value of `isLoadingReportData` was removed. This is causing regression in `withReportOrNotFound` to show `NotFoundView` page. In addition to other components using this HOC, my component was also using it and thus I was seeing the `NotFoundView` for few seconds in the beginning. I have added a fix for it in this PR as its was a one word fix and had to be added for my component to work correctly again.

Reply:
Looks like we're not using these language now. We can remove from en.ts and es.ts

Reply:
removed subtitle en.ts and es.ts